### PR TITLE
Create app.js

### PR DESCRIPTION
**Pull request by davidislam**
Screen Shot 2020-05-26 at 2 30 09 PM
#3 
The purpose of this ticket was to setup a web server using Node.js. I tested it in my browser by running node app.js in the terminal.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules
+node_modules
```
Comment (danielanatolie) [inline .gitignore:2]: Why is this ignored twice?

File: app.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const http = require("http");
```
Comment (danielanatolie) [inline app.js:1]: The file structure needs to be updated. Backend code should live in /backend/*

Comment (DavidAnatolie) [reply app.js]: d258c0b

File: .gitignore (hunk 1 above)
Comment (DavidAnatolie) [reply .gitignore]: d258c0b

**Comment by DavidAnatolie** [comment]
Screen Shot 2020-05-28 at 10 47 11 AM

d258c0b

**Pull request merged**